docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx

```diff
@@ -1,25 +1,96 @@
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
 import { Alert } from '@aws-amplify/ui-react';
 
+## 2.x to 3.x
+### Installation
+For Vue.js 2 users please continue to use our legacy libary found [here](https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy/amplify-ui-vue).
+
+Install the 3.x version of the `@aws-amplify/ui-vue` library.
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify @aws-amplify/ui-vue@3.x
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn add aws-amplify @aws-amplify/ui-vue@3.x
+```
+
+</TabItem>
+</Tabs>
+
+### Update and usage
+
+The following breaking changes were introduced in `@aws-amplify/ui-vue` major version `3.0`:
+
+#### Automatic signin on signup logic moved to `aws-amplify`. 
```

Comment:
Similar comment to the react migration guide. Let's add updating the libraries version to 5 above this heading, add a '1.' to this heading and remove the L5 heading below.

Reply:
Fixed